### PR TITLE
91 styling colors + spacing

### PR DESCRIPTION
**Pull request by Isebring**
## Problem

#91

## Solution / Implementation

Styled the whole page. 

## Testing

Added no tests in this branch sincewe were focusing on styling primarily.

### CODE REVIEW

File: index.html
Comment (lysmac) [inline index.html:1]: Good spot and fix!

File: src/components/header/Header.tsx
Comment (lysmac) [inline src/components/header/Header.tsx:1]: Great change from link to navlinks

File: src/routes/Bookmarks.tsx
Comment (lysmac) [inline src/routes/Bookmarks.tsx:1]: Good move to transfer the inline styles to a separate css-file 👏

File: src/tests/Bookmarks.test.tsx
Comment (lysmac) [inline src/tests/Bookmarks.test.tsx:1]: Keeping the tests up to date, good job!

**Review by lysmac — APPROVED** [review]

File: src/components/thumbnail/Thumbnail.css
Comment (ParhamInBinary) [inline src/components/thumbnail/Thumbnail.css:1]: Maybe add som gap between the thumbnails on the Y-axis to separate them for clarity.

**Review by ParhamInBinary — APPROVED** [review]
Looks great! Just checkout my suggestion before merging

File: src/components/thumbnail/Thumbnail.css
Comment (Isebring) [reply src/components/thumbnail/Thumbnail.css:1]: Thanks for the input! Looking into it now 👍

Comment (Isebring) [reply src/components/thumbnail/Thumbnail.css:1]: It is now fixed! 👍

**Pull request merged**